policy/modules/services/tuned.te

```diff
@@ -5,6 +5,27 @@ policy_module(tuned, 1.8.0)
 # Declarations
 #
 
+## <desc>
+## <p>
+## Control whether tuned can execute generated tmp files.
+## </p>
+## </desc>
+gen_tunable(tuned_exec_tmp_files,false)
+
+## <desc>
+## <p>
+## Control whether tuned can read raw memory.
+## </p>
+## </desc>
+gen_tunable(tuned_read_raw_memory,false)
```

Comment:
Please add a better descriptions, e.g. what features/settings will need these.

Reply:
I have communicated with the author.  He doesn't think the default profiles shipped with tuned shouldn't be executing temporary files that are created, but other profiles could.  I will try to work on these comments to make that clear.